src/document.rs

```diff
@@ -42,7 +42,7 @@ pub(crate) fn extract_value(any: &PyAny) -> PyResult<Value> {
     }
     if let Ok(datetime) = any.extract::<NaiveDateTime>() {
         return Ok(Value::Date(tv::DateTime::from_timestamp_secs(
-            datetime.timestamp(),
+            datetime.and_utc().timestamp(),
```

Comment:
This is unrelated to the PyO3. Just another warning from chrono that was easy to fix because the warning included the required change to resolve the deprecation warning.